app/views/content/what-is-a-pgce.md

```diff
@@ -19,28 +26,64 @@ keywords:
+A postgraduate certificate in education (PGCE) is an academic qualification. You can get this through teacher training.
 
-## PGCE
+You do not need one to be a qualified teacher in England, but it can help you develop your teaching skills and knowledge.
 
-The course you take to get a PGCE is a mixture of learning education theory, and school placements where you will be put in a classroom to learn from experience.
+## The qualifications you need to teach in England
 
-Entry requirements for a PGCE course usually include an undergraduate degree or an equivalent qualification. The course will usually take 1 year to complete full time, or 2 years part time.
+You need qualified teacher status (QTS) to teach in most primary, secondary and special schools in England. You can get QTS through undergraduate or postgraduate teacher training.
 
-[Find a postgraduate teacher training course](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+[Find out more about qualified teacher status (QTS)](/what-is-qts).
 
-### Types of PGCE courses
+Many teacher training courses also offer you a postgraduate qualification such as a PGCE, but you do not need one to teach.
 
-You can do a PGCE course at a university, or through a school-led teacher training programme.
+## Teacher training courses
 
-You can do a PGCE in primary education, secondary education, or further/adult education.
+You can do a teacher training course that leads to:
 
-## PGDE
+* just QTS
+* just PGCE
+* QTS with PGCE
 
-A postgraduate diploma in education (PGDE) course is very similar to a PGCE, the difference being the amount of academic credits you are awarded at the end of your course.
+Getting just QTS means that you’re qualified to teach in most schools in England. It may also mean that your course fees are lower, you submit fewer assignments, and the course could be shorter.
 
-## Teaching qualifications you need
+Getting just a PGCE means you will not be qualified to teach in most primary, secondary and special schools in England.
 
-If you want to be a primary school teacher or secondary school teacher in a state school in England, you will need to get qualified teacher status (QTS). You don’t need a PGCE or PGDE to become a teacher, but many courses offer it alongside QTS.
+[Read about Nathan who chose to get QTS with PGCE](/blog/salaried-teacher-training-classroom-learning).
 
-<a href="/what-is-qts" class="button">Find out more about qualified teacher status (QTS)</a>
+## Benefits of having a PGCE
+
+A PGCE can give you a better understanding of teaching practices, educational research and theory.
+
+It can also be beneficial if you want to teach in other countries because it’s an internationally recognised academic qualification. However, you should always check the local requirements of the country you’re looking to teach in.
+
+It can also award up to 60 credits at Master’s level which you could use towards a full Master’s qualification. This is usually 180 credits.
```

Comment:
I'm presuming this would be in the future and not linked to ITT? Do we need to make that clear?